api/fastpm/cosmology.h

```diff
@@ -12,6 +12,12 @@ struct FastPMCosmology {
     int N_nu;  // total number of neutrino species (massive and massless) (could use 'N_ur' as number of ur nus, instead, but dont want conflict with CLASS defn)
     double m_ncdm[3]; // masses of massive neutrinos (ncdm) for now assume max of 3 ncdm nus.
     int N_ncdm;
+    
+    /* Fermi Dirac interp for ncdm 
+       It's not really cosmology dependant,
+       but only cosmology uses it, so I guess
+       it makes sense to have a pointer to it here */
+    FastPMFDInterp * FDinterp;   //FIXME: I guess this will just be NULL in the case of no ncdm. Is this ugly?
```

Comment:
What if you make Cosmology a member of FastPMFDInterp? Then we don't have an optional member in cosmology that an be NULL  depending on magical global flags.